homework_2.py

```diff
@@ -0,0 +1,27 @@
+eleks_employees = ["Nataly", "John", "Peter", "James", "Jacob"]
+toshiba_employees = ["Nelly", "Harry", "Jimmy", "Natan", "Mark"]
+new_toshiba_employees_list = (toshiba_employees + eleks_employees)
```

Comment:
as I mentioned in the lesson, better to use functions not + for lists. union will be much better